pkg/pipelinerunmetrics/metrics.go

```diff
@@ -59,6 +61,16 @@ var (
 		"Number of pipelineruns executing currently",
 		stats.UnitDimensionless)
 	runningPRsCountView *view.View
+
+	runningPRsThrottledByQuotaCount = stats.Float64("running_pipelineruns_throttled_by_quota_count",
```

Comment:
I'm just curious, how useful is it to have separate pipelinerun metrics for this when they are derived from the taskruns?

Reply:
My intuition is to stay consistent w/ existing pod related metrics (we provide tekton_pipelines_controller_taskruns_pod_latency but not tekton_pipelines_controller_pipelineruns_pod_latency) and in favor of a smaller more focused PR; however if you feel it's helpful/convenient to have this at the pipelineRun level that's fine too! especially considering it's an experimental metric, we could always decide to remove it later if it's not helpful.

Reply:
no that comparison with pod latency resonates @lbernick 

I'll remove the pipelinerun related metric 

thanks